test/soca/gw/jjob_prep.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+bindir=$1
+srcdir=$2
+
+# export env. var.
+source ${srcdir}/test/soca/gw/runtime_vars.sh $bindir $srcdir
+
+# prepare background from previous cycle
```

Comment:
why is this being done here and not part of the prep job?

Reply:
Are you talking about the background for the previous cycle or the sourcing of the env. vars. @CoryMartin-NOAA ?

Reply:
the background, shouldn't it be 'available' already

Reply:
They would be after 1 cycle. But yes, good point ... It's not part of the prep jjob currently. I don't think I understand quite yet how the first cycle is handled in the g-w. I've been copying background files of the first cycle to a specific location to so far.

Reply:
I think you are talking about a 'half cycle' vs the 'full cycle', we shouldn't be testing the 'half cycle' because there is no DA yet. Assume there has already been a forecast, and the files you need are in `$COMROT` when you get to your prep step.

Reply:
Ha yes ... I forgot the jargon of the first cycle issue. 
So that's what that step above is doing, populating `$COMROT`, do you mean that step should be done in the previous test? Or hidden in the `CMakeLists.txt`?

Reply:
`$COMROT` should be `$GDASAPP_TESTDATA/lowres/` I think? Perhaps it needs slightly modified. Or I would suggest as you said a new test that just stages the ROTDIR.

Reply:
I see I wrote it's definitely not a ctest ... But I was convinced otherwise last week. It was added as a separate ctest. Marking as resolved @CoryMartin-NOAA .